src/test/script_tests.cpp

```diff
@@ -931,16 +931,31 @@ BOOST_AUTO_TEST_CASE(script_json_test)
     // amount (nValue) to use in the crediting tx
     UniValue tests = read_json(std::string(json_tests::script_tests, json_tests::script_tests + sizeof(json_tests::script_tests)));
 
+    const KeyData keys;
     for (unsigned int idx = 0; idx < tests.size(); idx++) {
         const UniValue& test = tests[idx];
         std::string strTest = test.write();
         CScriptWitness witness;
+        TaprootBuilder taprootBuilder;
         CAmount nValue = 0;
         unsigned int pos = 0;
         if (test.size() > 0 && test[pos].isArray()) {
             unsigned int i=0;
             for (i = 0; i < test[pos].size()-1; i++) {
-                witness.stack.push_back(ParseHex(test[pos][i].get_str()));
+                auto element = test[pos][i].get_str();
+                // Taproot script - third from the last element in witness stack
+                if (test[pos].size() >= 3 && i == test[pos].size()-3 && !IsHex(element)) {
+                    witness.stack.push_back(ToByteVector(ParseScript(element)));
+                // Taproot script control block - second from the last element in witness stack
+                // If <AUTOGEN:CONTROLBLOCK> we auto-generate the control block
+                } else if (test[pos].size() >= 3 && i == test[pos].size()-2 && strcmp(element.c_str(), "<AUTOGEN:CONTROLBLOCK>") == 0) {                    
```

Comment:
Removes whitespace that breaks linter
```suggestion
                } else if (test[pos].size() >= 3 && i == test[pos].size()-2 && strcmp(element.c_str(), "<AUTOGEN:CONTROLBLOCK>") == 0) {
```